cicd/forgeops-tests/lib/utils/am_pod.py

```diff
@@ -11,88 +11,71 @@
-    ROOT = os.path.join(os.sep, 'usr', 'local', 'tomcat')
-    TEMP = os.path.join(ROOT, 'fr-tmp')
+    REPRESENTATIVE_COMMONS_JAR_NAME = 'config'
 
-    def __init__(self, name, manifest_filepath):
+    def __init__(self, name):
         """
         :param name: Pod name
-        :param manifest_filepath: Path to product manifest file
         """
-        super().__init__(AMPod.PRODUCT_TYPE, name, manifest_filepath)
+        super().__init__(AMPod.PRODUCT_TYPE, name)
 
-    def is_expected_version(self):
-        """
-        Return True if the version is as expected, otherwise assert.
-        :return: True if the version is as expected.
-        """
+    def get_version(self):
+        """Get the application's version."""
 
         amcfg = AMConfig()
 
-        logger.info('Get admin token')
+        logger.debug('Get admin token')
         headers = {'X-OpenAM-Username': 'amadmin', 'X-OpenAM-Password': 'password',
                    'Content-Type': 'application/json', 'Accept-API-Version': 'resource=2.0, protocol=1.0'}
 
         response = post(verify=amcfg.ssl_verify, url=amcfg.rest_authn_url, headers=headers)
         rest.check_http_status(http_result=response, expected_status=200)
         admin_token = response.json()['tokenId']
 
-        logger.info('Get AM version')
+        logger.debug('Get AM version')
         headers = {'Content-Type': 'application/json', 'Accept-API-Version': 'resource=1.0',
                    'iplanetdirectorypro': admin_token}
         response = get(verify=amcfg.ssl_verify, url=amcfg.am_url + '/json/serverinfo/version', headers=headers)
         rest.check_http_status(http_result=response, expected_status=200)
 
-        logger.info('Check AM version')
-        assert response.json()['version'] == self.manifest['version'], 'Unexpected AM version'
-        assert response.json()['revision'] == self.manifest['revision'], 'Unexpected AM build revision'
-        assert response.json()['date'] == self.manifest['date'], 'Unexpected AM build date'
-        return True
+        version = response.json()['version']
+        revision = response.json()['revision']
+        date = response.json()['date']
+        am_metadata = {'TITLE': self.product_type,
+                       'DESCRIPTION': self.name,
+                       'VERSION': version,
+                       'REVISION': revision,
+                       'DATE': date}
 
-    def setup_commons_check(self, namespace):
-        """
-        Setup for checking commons library version
-        :param namespace The kubernetes namespace.
-        """
+        return am_metadata
 
-        logger.debug('Setting up for commons version check')
-        config_version_jar = 'config-%s.jar' % self.manifest['commons_version']
-        test_jar_filepath = os.path.join(AMPod.ROOT, 'webapps', 'am', 'WEB-INF', 'lib', config_version_jar)
-        super(AMPod, self).setup_commons_check(namespace, test_jar_filepath, AMPod.TEMP)
+    def is_expected_version(self):
```

Comment:
As we don't compare the version to an expected one maybe we should rename the method to `get_version()` or `print_version()` something like that?